shiny/ui/_theme_brand.py

```diff
@@ -0,0 +1,297 @@
+from __future__ import annotations
+
+import re
+import warnings
+from pathlib import Path
+from typing import Any, Optional
+
+from brand_yml import Brand
+from htmltools import HTMLDependency
+
+from .._versions import bootstrap
+from ._theme import Theme
+from ._theme_presets import ShinyThemePreset, shiny_theme_presets
+
+color_map = {
```

Comment:
Should more strongly type this since, when we use this value, we are making fairly strong assumptions about the items

```suggestion
color_map: dict[str, list[str]] = {
```